scripts/airdrop.js

```diff
@@ -21,13 +24,55 @@ const argv = default_yargs
     describe: "Do not actually send transactions, just simulate their submission",
   }).argv
 
+const parseCsvFile = async function (filePath) {
```

Comment:
I think this belongs more in a utils file, right?
Is not related to airdrop and can be used with any CSV

Reply:
Yes, I suppose so.